bob-grpc/proto/bob.proto

```diff
@@ -89,5 +89,8 @@ message BobError {
 
 // Delete operation options
 message DeleteOptions {
-    bool force_node = 1; // Force operation to be served by node to which it comes
+    repeated string remote_nodes = 1;
+    bool force_node = 2; // Force operation to be served by node to which it comes
+    bool force_delete = 3; // Force add record marker to storage
+    uint64 timestamp = 4; // User level timestamp
```

Comment:
It is better to add `timestamp` to `DeleteRequest` directly. Also, I think it's even better to use a `BlobMeta` message